docker/ambulance-analytics-sandbox/src/mocks/submissionResponse.json

```diff
@@ -0,0 +1,3 @@
+{
+    "id": "6eeec54e-3d39-4bc4-a1d9-57615d258fpq"
```

Comment:
Might be better to return the "Nil" UUID for the sandbox response, to indicate this is a test server. This UUID looks genuine - users might conflate some significances to the UUID, whereas the nil uuid is obvious its an example: "00000000-0000-0000-0000-000000000000"